setup.py

```diff
@@ -59,9 +61,11 @@ def main():
         license='Apache2.0',
         url='https://github.com/google/mobly',
         download_url='https://github.com/google/mobly/tarball/1.8.1',
-        packages=setuptools.find_packages(),
+        package_dir={'mobly':'mobly'},
+        packages=setuptools.find_packages(exclude=['tests']),
```

Comment:
I think these two lines are sufficient.
I don't think we need to do the absolute path thing?

Reply:
Actually I'm still seeing the "tests" files included in the distribution with this change when using cmd:
```
$ python  setup.py sdist --formats=zip
```

Reply:
When I generate this, I'm doing it within a virtualenv. Are you sure it isn't already a part of your PYTHONPATH/installed?

Here are the full steps I am using:

```
$ mobly_dir="~/mobly"
$ python3 -m virtualenv /tmp/venv >&2 >/dev/null
$ source /tmp/venv/bin/activate
$ cp -r $mobly_dir /tmp/venv/
$ cd /tmp/venv/mobly/
$ python3 setup.py sdist --formats=zip
...
$ python3 -m easy_install dist/mobly-1.8.1.zip
...
$ cd / # Somewhere not next to the tests/ folder in the mobly base directory
$ python3.5
Type "help", "copyright", "credits" or "license" for more information.
>>> import mobly
>>> import tests
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named 'tests'
>>> 
```
I confirmed this works with only making a single line change. I'll update the PR now.

Reply:
I see.
Seems like the tests will be added to sdist regardless but when someone installs from that zip, it won't install the tests.

Reply:
Change is up. Confirmed installation from the zip does not install tests.